### PR TITLE
add initial readme

### PR DESCRIPTION
**Pull request by TomasTomecek**
Done
- [x] intro
- [x] first batch of roles

TODO (for next PR)
- [ ] alerting role
- [ ] release role

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+operations and is not disrupted when a key member leaves for a vacation.
+
+This repository is an evolution of the rotating roles that we started in the Packit team. You can see it in action [here](https://github.com/packit/agile) and check role assignments in the [repository's issues](https://github.com/packit/agile/issues).
+
+After the technique proved successful, we followed with a series of blog posts and presentations:
+1. 📄 [How to run an open-source service?](https://medium.com/@laura.barcziova/how-to-run-an-open-source-service-fb3303240e69)
+2. 📄 [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
+3. 📄 [Share team responsibilities efficiently](https://medium.com/@laura.barcziova/share-team-responsibilities-efficiently-9a202aad7bd0)
+4. 📄 [Role rotation tutorial](https://medium.com/@laura.barcziova/role-rotation-tutorial-957ed3545ef2)
+5. 📽️ [A journey of iteration and role rotation for better collaboration - DevConf.CZ 2024](https://www.youtube.com/watch?v=y1t7Wd31bL8)
```
Comment (lbarcziova) [inline README.md]:
```suggestion
After the technique proved successful, we followed with a series of blog posts:
1. 📄 [How to run an open-source service?](https://medium.com/@laura.barcziova/how-to-run-an-open-source-service-fb3303240e69)
2. 📄 [Inception of rotating roles](https://medium.com/@laura.barcziova/inception-of-rotating-roles-9caf971b3096)
3. 📄 [Share team responsibilities efficiently](https://medium.com/@laura.barcziova/share-team-responsibilities-efficiently-9a202aad7bd0)
4. 📄 [Role rotation tutorial](https://medium.com/@laura.barcziova/role-rotation-tutorial-957ed3545ef2)

And we also did a presentation:
- 📽️ [A journey of iteration and role rotation for better collaboration - DevConf.CZ 2024](https://www.youtube.com/watch?v=y1t7Wd31bL8)
```

Comment (lbarcziova) [reply README.md]: maybe just my preference, to have it split

File: README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+## 🗓️Meeting lead
+Having a productive meeting that sticks to agenda, has notes, and a clear
+outcome, has a tremendeous impact on everyone. With one responsible meeting
+lead, you can stop wasting time of all people on your meetings.
+
+### Responsible for
+* Run a meeting (TBD: this could be more detailed)
+* Make sure people stay on topic
+* Follow meeting agenda - address all points
+* Write down action items and assign them
```
Comment (lbarcziova) [inline README.md]:
```suggestion
* Write down action items and make sure the they are assigned

File: README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+* Well run meeting is the difference between a complete waste of time and
+  effectively coordinating in a group of people
+
+### What can people learn?
+* Not everyone likes the attention, so this can be a big step outside of
+  comfort zone for many.
+* It can be tiring to be in charge of everything all the time. Managers
+  understand. Team members may suddenly feel more empathy and compassion once
+  they realize that leading a meeting requires significant effort.
+* Better knowledge about the code base to be able to review the changes.
```
Comment (lbarcziova) [inline README.md]:
```suggestion

```
this is probably by mistake taken from the review role

File: README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+
+## 🏃 Agile lead
+As we participate in our agile methodology technique, be it scrum or kanban, we
```
Comment (lbarcziova) [inline README.md]:
```suggestion
As we participate in our agile methodology technique, be it scrum, kanban, or anything else, we
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+should have someone to be in charge. This role can be very similar to the
+Meeting lead. We are keeping it separate here so we can clearly distinguish
+between agile development methodology and just running meetings.
+
+### Responsible for
+* The team sticks to all the agile ceremonies as they are defined.
+* Coordinate with the meeting lead are be one.
```
Comment (lbarcziova) [inline README.md]:
```suggestion
* Coordinate with the meeting lead or be one.
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+* Assist people who are struggling in their assignments.
+* Volunteer if possible for i.e. research, testing, or reaching out to
+different teams if needed.
+* Celebrate success with the team.
+
+### Why have this role?
+* It could become chaotic without an agile lead.
+
+### What can people learn?
+* The responsibility to be in charge of the team's sprint.
```
Comment (lbarcziova) [inline README.md]: I would prefer not explicitly writing `sprint`, but not sure how to generalise it :thinking:

File: README.md [hunk 7]
```diff
@@ -0,0 +1,141 @@
...
+often not taught at a school. When users can directly interact with project's
+developers, it allows them to quickly address issues and start a lasting
+professional relationship.
+
+This role can be paired with the watcher of alerts and monitoring.
+
+### Responsible for
+* Pay attention to questions and comments from your community in your chat
+rooms, issue trackers, mailing lists, social media.
+* Bring user's pain points back to the team and make sure they are prioritized.
```
Comment (lbarcziova) [inline README.md]:
```suggestion
* Bring user's pain points back to the team and make sure they are triaged and prioritized.
```

**Review by lbarcziova — APPROVED** [review]
Great job! Just few smallish comments.

File: README.md (hunk 1 above)
Comment (TomasTomecek) [reply README.md]: yup, this is indeed more readable

File: README.md (hunk 3 above)
Comment (TomasTomecek) [reply README.md]: oh my, you are right, I placed it incorrectly; thank you!

File: README.md (hunk 4 above)
Comment (TomasTomecek) [reply README.md]: I wonder what other methodologies are being used actually :D

File: README.md (hunk 6 above)
Comment (TomasTomecek) [reply README.md]: yep, that's exactly what I was struggling with

Comment (lbarcziova) [reply README.md]: maybe something like iteration? or team's work for dedicated time unit :D but then `sprint` might be better in the end

Comment (TomasTomecek) [reply README.md]: I like iteration, that's a good suggestion.

**Review by lbarcziova — APPROVED** [review]
:rocket: :rocket:

**Pull request merged**